web/web/src/lib/utils/initial.js

```diff
@@ -308,5 +308,33 @@ export const providers = [
         description: 'Apache Hudi catalog uri config'
       }
     ]
+  },
+  {
+    label: 'OceanBase',
+    value: 'jdbc-oceanbase',
+    defaultProps: [
+      {
+        key: 'jdbc-driver',
+        value: '',
+        required: true,
+        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver'
+      },
+      {
+        key: 'jdbc-url',
+        value: '',
+        required: true,
+        description: 'e.g. jdbc:mysql://localhost:3306'
```

Comment:
The default sql port of OceanBase is 2881.

Reply:
fixed